file-formats/dcat/dcat-v1.json

```diff
@@ -66,7 +66,18 @@
           "title": "Sensitivity Level",
           "description": "Sensitivity level for data category",
           "type": "string",
-          "maxLength": 1
+          "enum": [
+            "sensitivePersonalData",
+            "businessSecret"
+          ],
+          "enumTitles": [
+            "Sensitive Personal Data",
+            "Business Secret"
+          ],
+          "enumDescriptions": [
+            "Sensitive Personal Data",
```

Comment:
Be aware, that also the descriptions of enums supposed to be `Sentence case`. Change to 'Sensitive personal data' and 'Business secret' unless the terms are found in sapterm.com

Reply:
fixed